driver_cpl/cime_config/buildexe

```diff
@@ -20,10 +20,10 @@ logger = logging.getLogger(__name__)
 def _main_func():
 ###############################################################################
 
-    caseroot, libroot, _ = parse_input(sys.argv)
+    caseroot, bldroot, libroot = parse_input(sys.argv)
```

Comment:
Are you sure this order is correct? The `parse_input` function has a return that looks like:

```
    return args.caseroot, args.libroot, args.bldroot
```

but you have bldroot and libroot swapped. If this is truly correct, please add a comment explaining this unintuitive swapping.